libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -782,6 +785,114 @@ singleton AP_EFI method get_state void EFI_State'Ref
+userdata AP_ExternalAHRS::gps_data_message_t field horizontal_pos_accuracy float'skip_check write
+userdata AP_ExternalAHRS::gps_data_message_t field vertical_pos_accuracy float'skip_check write
+userdata AP_ExternalAHRS::gps_data_message_t field hdop float'skip_check write
+userdata AP_ExternalAHRS::gps_data_message_t field vdop float'skip_check write
+userdata AP_ExternalAHRS::gps_data_message_t field longitude int32_t'skip_check write
+userdata AP_ExternalAHRS::gps_data_message_t field latitude int32_t'skip_check write
+userdata AP_ExternalAHRS::gps_data_message_t field msl_altitude int32_t'skip_check write
+userdata AP_ExternalAHRS::gps_data_message_t field ned_vel_north float'skip_check write
+userdata AP_ExternalAHRS::gps_data_message_t field ned_vel_east float'skip_check write
+userdata AP_ExternalAHRS::gps_data_message_t field ned_vel_down float'skip_check write
+
+userdata AP_ExternalAHRS::ins_data_message_t depends (AP_EXTERNAL_AHRS_SCRIPTING_ENABLED == 1)
+userdata AP_ExternalAHRS::ins_data_message_t rename ins_data_message_t
+userdata AP_ExternalAHRS::ins_data_message_t field accel Vector3f write
+userdata AP_ExternalAHRS::ins_data_message_t field gyro Vector3f write
+userdata AP_ExternalAHRS::ins_data_message_t field temperature float'skip_check write
+
+userdata AP_ExternalAHRS::airspeed_data_message_t depends (AP_EXTERNAL_AHRS_SCRIPTING_ENABLED == 1)
+userdata AP_ExternalAHRS::airspeed_data_message_t rename airspeed_data_message_t
+userdata AP_ExternalAHRS::airspeed_data_message_t field differential_pressure float'skip_check write
+userdata AP_ExternalAHRS::airspeed_data_message_t field temperature float'skip_check write
+
+singleton AP_ExternalAHRS method handle_scripting boolean AP_ExternalAHRS::state_t nav_filter_status_flags_t
+singleton AP_InertialSensor method handle_external void AP_ExternalAHRS::ins_data_message_t
+singleton AP_InertialSensor method handle_external depends (AP_EXTERNAL_AHRS_SCRIPTING_ENABLED == 1)
+singleton AP_Baro method handle_external void AP_ExternalAHRS::baro_data_message_t
+singleton AP_Baro method handle_external depends (AP_EXTERNAL_AHRS_SCRIPTING_ENABLED == 1)
+singleton AP_GPS method get_first_external_instance boolean uint8_t'Null
+singleton AP_GPS method get_first_external_instance depends (AP_EXTERNAL_AHRS_SCRIPTING_ENABLED == 1)
+singleton AP_GPS method handle_external void AP_ExternalAHRS::gps_data_message_t uint8_t'skip_check
+singleton AP_GPS method handle_external depends (AP_EXTERNAL_AHRS_SCRIPTING_ENABLED == 1)
+singleton Compass method handle_external void AP_ExternalAHRS::mag_data_message_t
+singleton Compass method handle_external depends (AP_EXTERNAL_AHRS_SCRIPTING_ENABLED == 1)
+
+include AP_Airspeed/AP_Airspeed.h
+singleton AP_Airspeed depends (AP_AIRSPEED_EXTERNAL_ENABLED == 1) && (APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI)
+singleton AP_Airspeed semaphore
+singleton AP_Airspeed rename airspeed
```

Comment:
I'm amazed `airspeed` is not used a local variable in any of our example scripts. But this might conflict with scripts in the wild.

I think its OK because the local variable with overwrite the global so the script will work, just not the this new lib, but it might be worth checking, would also be interesting to see if luacheck would catch it.